app_list.js

```diff
@@ -20,7 +20,7 @@ app.controller("app_list", function($scope, $http) {
         });
         
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/app_list.js
+++ b/app_list.js
@@ -18,7 +18,7 @@
             queryTokenizer: Bloodhound.tokenizers.whitespace,
             local: suggestionList
         });
-        
+
         // Initializing the typeahead
 
         $('.typeahead').typeahead({
```